api/src/paths/dwc/eml.ts

```diff
@@ -0,0 +1,918 @@
+    emlRoot.dataset.project.personnel = {
+      individualName: { givenName: survey.lead_first_name, surName: survey.lead_last_name },
+      organizationName: project.coordinator_agency_name,
+      role: 'pointOfContact'
+    };
+  } else {
+    emlRoot.dataset.project.personnel = {
+      organizationName: organizationFullName,
+      onlineUrl: organizationURL,
+      role: 'custodianSteward'
+    };
+  }
+
+  emlRoot.dataset.project.abstract = { section: { title: 'Objectives', para: survey.objectives } };
+
+  if (surveyFundingSource.length) {
+    emlRoot.dataset.project.funding = getFundingEML(surveyFundingSource);
+  }
+
+  emlRoot.dataset.project.studyAreaDescription = { coverage: {} };
+  const surveyGeographicDescription = survey.location_description
+    ? survey.location_name + ' - ' + survey.location_description
+    : survey.location_name;
+  emlRoot.dataset.project.studyAreaDescription.coverage.geographicCoverage = getGeographicCoverageEML(
+    surveyGeographicDescription,
+    surveyBoundingBox,
+    surveyPolygons
+  );
+
+  emlRoot.dataset.project.studyAreaDescription.coverage.temporalCoverage = getTemporalCoverageEML(survey);
+
+  emlRoot.dataset.project.studyAreaDescription.coverage.taxonomicCoverage = { taxonomicClassification: [] };
+  focalTaxonomicCoverage.rows.forEach(function (row: any, i: number) {
+    emlRoot.dataset.project.studyAreaDescription.coverage.taxonomicCoverage.taxonomicClassification[i] = {
+      taxonRankName: row.tty_name,
+      taxonRankValue: row.unit_name1 + ' ' + row.unit_name2,
+      commonName: row.english_name,
+      taxonId: { $: { provider: taxonomicProviderURL }, _: row.code }
+    };
+  });
+
+  emlRoot.dataset.project.relatedProject = { $: { id: project.uuid, system: simsProviderURL }, title: project.name };
+  if (project.coordinator_public) {
+    emlRoot.dataset.project.relatedProject.personnel = {
+      individualName: { givenName: project.coordinator_first_name, surName: project.coordinator_last_name },
+      organizationName: project.coordinator_agency_name,
+      electronicMailAddress: project.coordinator_email_address,
+      role: 'pointOfContact'
+    };
+  } else {
+    emlRoot.dataset.project.relatedProject.personnel = {
+      organizationName: organizationFullName,
+      onlineUrl: organizationURL,
+      role: 'custodianSteward'
+    };
+  }
+
+  emlRoot.dataset.project.relatedProject.abstract = {
+    section: [
+      { title: 'Objectives', para: project.objectives },
+      { title: 'Caveats', para: project.caveats },
+      { title: 'Comments', para: project.comments }
+    ]
+  };
+
+  if (projectFundingSource.length) {
+    emlRoot.dataset.project.relatedProject.funding = getFundingEML(projectFundingSource);
+  }
+
+  emlRoot.dataset.project.relatedProject.studyAreaDescription = { coverage: {} };
+  emlRoot.dataset.project.relatedProject.studyAreaDescription.coverage.geographicCoverage = getGeographicCoverageEML(
+    'Not Supplied',
```

Comment:
Is this meant to be hardcoded? 

(noticed that when I added a project boundary description it didnt show up int he eml)

Reply:
nice catch. will address.